### ISSUE TITLE
force GUI on stop matching

### ISSUE THREAD

**Issue by koenvanhollebeke** [Gabboxl/gtfs-osm-import#18]
Hi,
It would be nice to have to be able to force the stop matching GUI to always on for debugging purposes. Something like the interactivedebug mode or an option inside.
KR
Koen

**Comment by Gabboxl** [comment]
Could you explain when should the GUI be always shown? Currently it is shown only when it is required to choose the right coordinates between GTFS and OSM stops.

**Comment by koenvanhollebeke** [comment]
> Oh alright, I think I understand your request now: you would like to have a GUI that shows potential matches like gtfs-osm-sync, am I right?
Yes - for debug. It is already there in your code, just make it possible to view it even if in normal automatic operation it doesn't needs to show up.

**Comment by Gabboxl** [comment]
> Yes - for debug. It is already there in your code, just make it possible to view it even if in normal automatic operation it doesn't needs to show up.

Okay, alright. A simple GUI that shows stops that would be matched/removed/added I think can be easy to implement. Will look into it.
